package.json

```diff
@@ -19,7 +20,8 @@
     "ws": "^3.0.0"
   },
   "scripts": {
-    "compile": "tsc",
+    "clean": "rimraf dist",
```

Comment:
Noticed that even when I removed the old `.ts` files, it would still include old test files when I ran `npm test`. Added this `clean` script to run before compile to make sure we aren't accidentally publishing/testing deleted files that may have snuck in.